tests/cli/PdxVersion1Lib/PdxVersion1Lib.csproj.in

```diff
@@ -109,7 +109,7 @@
     <ProjectReference Include="$(CMAKE_BINARY_DIR)\clicache\src\Apache.Geode.vcxproj">
       <CopyLocalSatelliteAssemblies>true</CopyLocalSatelliteAssemblies>
       <ReferenceOutputAssembly>true</ReferenceOutputAssembly>
-    </ProjectReference>
+    </ProjectReference>    
```

Comment:
Do we really need to add this whitespace here and the next file. If not then these two files would have no changes.